tests/cypress/page-object/languageSettings.ts

```diff
@@ -0,0 +1,13 @@
+export class LanguageSettings {
+    static visit(siteKey: string, language: string): void {
+        cy.visit(`/cms/editframe/default/${language}/sites/${siteKey}.manageLanguages.html`)
+    }
+
+    public SwitchReplaceUntranslatedWithDefault() {
```

Comment:
We usually set a lowercase for the first letter of functions can you please change to switchReplaceUntranslatedWithDefault? 
And add the return value to : void. I think you have a lint issue here